### ISSUE TITLE
RuntimeError: Empty probability map found (None)

### ISSUE THREAD

**Issue by meghbhalerao** [fepegar/torchio#384]
Hi all,
I am getting this error: `RuntimeError: Empty probability map found (None)` when I use `LabelSampler` during training. From the error, it seems like the **labelmap** for the image has only zeros. Is there any way to see which image is giving this issue..because I am getting this issue when I do the first `next(train_loader)` always. So, I am not able to check which Labelmap is zero.
Thanks for your help,
Megh

**Comment by fepegar** [comment]
Hi, @meghbhalerao. Thanks for reporting. Please next time include some code, it would help to see how you instantiate the sampler.

I have been able to reproduce:

```python
In [1]: import torchio as tio
   ...: 
   ...: fpg = tio.datasets.FPG()
   ...: fpg.seg.data *= 0
   ...: 
   ...: s = tio.LabelSampler(10)
   ...: next(s(fpg))
```

```python-traceback
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-1-2fdd200dba7f> in <module>
      5 
      6 s = tio.LabelSampler(10)
----> 7 next(iter(s(fpg)))

~/git/torchio/torchio/data/sampler/weighted.py in __call__(self, subject, num_patches)
     69             raise RuntimeError(message)
     70         probability_map = self.get_probability_map(subject)
---> 71         probability_map = self.process_probability_map(probability_map)
     72         cdf = self.get_cumulative_distribution_function(probability_map)
     73 

~/git/torchio/torchio/data/sampler/weighted.py in process_probability_map(self, probability_map)
    109                 f' ({self.probability_map_name})'
    110             )
--> 111             raise RuntimeError(message)
    112         data /= total  # normalize probabilities
    113         return data

RuntimeError: Empty probability map found (None)
```

I have modified the error message to print the problematic image path:

```python-traceback
Traceback (most recent call last):
  File "ll.py", line 7, in <module>
    next(s(fpg))
  File "/home/fernando/git/torchio/torchio/data/sampler/weighted.py", line 72, in __call__
    probability_map = self.process_probability_map(probability_map, subject)
  File "/home/fernando/git/torchio/torchio/data/sampler/weighted.py", line 116, in process_probability_map
    raise RuntimeError(message)
RuntimeError: Empty probability map found: /home/fernando/.cache/torchio/fpg/t1_seg_gif.nii.gz
```

**Comment by meghbhalerao** [comment]
Hi @fepegar, 
Thanks a lot! This is really helpful!
Megh

**Comment by meghbhalerao** [comment]
Hi @fepegar,
Just wanted to know if I do `pip install torchio`, will this change be reflected?
Thanks,
Megh

**Comment by fepegar** [comment]
If `torchio` is already installed, you'll need to add the `--upgrade` flag: `pip install --upgrade torchio`.

**Comment by fepegar** [comment]
(This fix should be available in `v0.18.8`)

**Comment by meghbhalerao** [comment]
Thank you so much!
